src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkShippingLocation.tsx

```diff
@@ -42,29 +64,95 @@ export const SubmitArtworkShippingLocation = () => {
     },
   })
 
+  console.log(values.location?.countryCode)
   return (
     <ProvideScreenTrackingWithCohesionSchema
       info={screen({
         context_screen_owner_type: OwnerType.submitArtworkStepShippingLocation,
         context_screen_owner_id: values.submissionId || undefined,
       })}
     >
-      <Flex px={2} flex={1}>
-        <ScrollView>
-          <Flex>
+      <ScrollView contentContainerStyle={{ paddingBottom: 50 }}>
+        <Flex px={2} flex={1}>
```

Comment:
Would this work as well?

```suggestion
      <ScrollView>
        <Flex px={2} pb={4} flex={1}>
```

Reply:
yes, this should work as well. I added it to the scrollview though because I feel like it belongs there as an extra space for scrolling instead of the Flex. If someone would accidentally add a new `Flex` below it, it would break the extra padding for space. We do the same in few other areas as well

Reply:
That makes a lot of sense 👍